src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    protected String color = ColorSupplier.getRandomColor();
```

Comment:
it isn't correct to use ColorSupplier in the Figure class
let's remove it from here